src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(
```

Comment:
I am almost positive this file is auto generated this way when you run `npx create-react-app` with the `--template minimal` flag, but as of React 18, `ReactDOM.render()` is no longer supported.

You can change this file to instead use the new `createRoot` function, like so: 

```js
import { createRoot } from 'react-dom/client';

const container = document.getElementById('root');
const root = createRoot(container);

root.render(<App />);
```

📖 If you'd like to read more about this change, you can check out this blog post from React: [Updates to Client Rendering APIs](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis)